client/test/e2e/cash/cash.spec.js

```diff
@@ -80,36 +86,38 @@ describe('Cash Payments Module', function () {
       browser.get(path);
 
       // the browser should be rerouted to the cashboxB page
-      expect(helpers.getCurrentPath()).to.eventually.equal(target);
+      expect(helpers.getCurrentPath()).to.eventually.equal(path);
     });
 
     it('should allow a user to select and deselect a cashbox', function () {
+      // the auxiliary cashbox is the target
```

Comment:
Tomorrow we will discuss how this routing should work.  This is close, but we should make sure the behavior is well-defined.